services/galley/galley.integration.yaml

```diff
@@ -37,13 +37,11 @@ settings:
     # able to register new idps.
     # disabled for integration tests (the ones who need it on will
     # turn it on themselves).
-    sso: false
+    sso: disabled-by-default
```

Comment:
Same comments apply here than to https://github.com/wireapp/wire-server-deploy/pull/144:

* specify in comments the allowed options
* update the comment above.